docs/getting-started/github-apps/github-apps.rst

```diff
@@ -66,6 +66,29 @@ to associate with a workflow on Dockstore. A template for both workflows and ser
 along with explanations for each field. For every branch/release on GitHub that has one of these files, a corresponding entry
 will be made on Dockstore.
 
+Error Handling
+----------------------------------
+Since Dockstore relies on GitHub to tell us when changes have been made on GitHub, there are chances that the message gets lost or delayed.
+Typically, Dockstore reacts within seconds of a change being made on GitHub, however service disruptions can delay this to a few minutes.
```

Comment:
"within seconds" -> "within a couple of minutes". Last I checked, it was about 90 seconds, 2 minutes leaves us some buffer room.

GitHub does notify us right away I believe, but we have an intentional delay in there due to some issues we ran into. We should not set expectations that the users will see changes in Dockstore almost immediately.

Reply:
I would just drop the service disruptions section.